fetchcode/api.py

```diff
@@ -0,0 +1,55 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import tempfile 
+
+import requests
+
+
+class Response:
+    """
+    this object stores all the information regarding the URL which our API
+    has hit 
+    1) Location of file
+    2) Content type of the file
+    3) Size of the file
+    4) URL of the file
+    """
```

Comment:
What about using this instead:
```
Represent the response from fetching a URL with:
- `filename`: the absolute location of the files that was fetched
- `content_type`: content type of the file
- `size`: size of the retrieved content in bytes
- `url`: fetched URL
```

Also in hindsight, what about using `location` and not `filename` and ensure this is an absolute file location? and not just a file name?

Reply:
All changes have been done, please have a look

Reply:
:+1: